marlowe-playground-client/src/Examples/JS/Contracts.purs

```diff
@@ -504,7 +504,7 @@ contractForDifferencesWithOracle =
 
     function recordEndPrice(name: ValueId, choiceId1: ChoiceId, choiceId2: ChoiceId,
         continuation: Contract): Contract {
-        return Let(name, DivValue(MulValue(priceBeginning, MulValue(ChoiceValue(choiceId1), ChoiceValue(choiceId2))), (Constant 10_000_000_000_000_000n)),
+        return Let(name, DivValue(MulValue(priceBeginning, MulValue(ChoiceValue(choiceId1), ChoiceValue(choiceId2))), (Constant (10_000_000_000_000_000n))),
```

Comment:
What does this do?

Reply:
It stores a value in the state to be used later to calculate the difference. The particular change was to add some missing parentheses in the Constant constructor